gluoncv/auto/estimators/base_estimator.py

```diff
@@ -11,6 +11,7 @@
 from ...utils import random as _random
 from ...utils.filesystem import temporary_filename
 from .utils import _suggest_load_context
+from autogluon.core.constants import MULTICLASS, BINARY, REGRESSION
```

Comment:
This will essentially make gluoncv depend on AutoGluon. I'm not sure if it is appropriate. @zhreshold

Reply:
One solution is to manually specify the MULTICLASS / BINARY / REGRESSION strings in the base_estimator file.

Reply:
Also, it may not be a good idea to touch the base_estimator. We can just change the image_classification class or rename it to image predictor.

Reply:
put this import inside of `_fit` function so it's won't affect gluoncv's other modules. For the auto module it's okay to import autogluon core module.

Reply:
@sallypannn you can have the problem_type in ImageClassificationEstimator but not the base estimator, it's probably too early to determine the problem types for e.g., ObjectDetection or SemanticSegmentation.